transfer-data-researchdrive.md

```diff
@@ -39,18 +40,80 @@ your PI:
     > ## Note about NetIDs
     > If your CHTC account is not tied to your campus NetID or you are accessing a data 
     > storage service that doesn't use your NetID, you'll need to omit the `-k` flag above
-4. **Choose a folder, part 2:** If you type `ls` now, you'll see the files in Research Drive, not CHTC. 
-Navigate through Research Drive (using `cd`) until you are at the folder where you would 
+4. **Choose a folder, part 2:** If you type `ls` now, you'll see the files in ResearchDrive, not CHTC. 
+Navigate through ResearchDrive (using `cd`) until you are at the folder where you would 
 like to get or put files. 
 5. **Move files:** To move files, you will use the `get` and `put` commands: 
-    - To move files from CHTC to Research Drive, run: 
+    - To move files from CHTC to ResearchDrive, run: 
         ```
         smb: \> put filename
         ```
         {: .term}
-    - To move files from Research Drive to CHTC, run: 
+    - To move files from ResearchDrive to CHTC, run: 
         ```
         smb: \> get filename
         ```
         {: .term}
-6. **Finish:** Once you are done moving files, you can type `exit` to leave the connection to Research Drive. 
+6. **Finish:** Once you are done moving files, you can type `exit` to leave the connection to ResearchDrive. 
+
+## Transferring A Batch of Files
+
+The steps described above work well for transferring a single file, or tar archive of 
+many files, at a time and is best for transferring a few files in a session. However 
+`smbclient` also provides options for transferring many individual files in a single command 
+using the `*` wildcard character.
+
+To transfer multiple files at once, first you must turn off `smbclient` file transfer prompt, 
+then use either `mget` or `mput` for your file transfer. For example, if you have multiple `fastq.gz` 
+to transfer to `/staging`:
+
+1. **Log in:** Log into the transfer server, `transfer.chtc.wisc.edu`. 
+2. **Choose a folder:** Navigate to the folder in `/staging` where you would like transfer your files to. 
+3. **Connect to ResearchDrive:** Run the following command to connect to ResearchDrive, filling in the username of 
+your PI: 
+    ```
+    [alice@transfer]$ smbclient -k //research.drive.wisc.edu/PI-Name
+    ```
+    {: .term}
+4. **Navigate to appropriate ResearchDrive directory**
+    ```
+    smb: \> cd path/to/files
+    ```
+    {: .term}
+5. **Turn of Prompting**
+    ```
+    smb: \> prompt
+    ```
+    {: .term}
+6. **Use `mget` instead of `get`**
+    ```
+    smb: \> mget *.fastq.gz
+    ```
+    {: .term}
+    
+As another example, use `smbclient` to transfer multiple `tar.gz` output files 
```

Comment:
I think we could condense these examples but this is fine for now.

Reply:
we chatted more and decided that we can condense in a subsequent PR